src/rust/include/rust/constants.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+#ifndef ZCASH_RUST_INCLUDE_RUST_CONSTANTS_H
+#define ZCASH_RUST_INCLUDE_RUST_CONSTANTS_H
+
+#include <stddef.h>
+
+#ifdef __cplusplus
+namespace blake2b {
+    const size_t PERSONALBYTES = 16;
```

Comment:
nit: a comment pointing to the issue in `cxx` for adding constants so someone will realize when we can remove this file.